src/main/java/calculator/ui/FormulaInput.java

```diff
@@ -0,0 +1,55 @@
+package calculator.ui;
+
+import calculator.util.FormulaUtil;
+import calculator.util.Operation;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FormulaInput {
+
+    public static List<String> getFormula() {
+        String input = "";
+        List<String> formular = new ArrayList<>();
+
+        System.out.println("계산할 연산식을 입력해주세요.");
+        try (BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in))) {
+            input = bufferedReader.readLine();
```

Comment:
```suggestion
            String input = bufferedReader.readLine();
```
중요한건 아니지만 14라인이 없어도 괜찮을 것 같아요 😃

Reply:
오호~반영했습니다!